mobile/mysterium/openvpn_connection_setup.go

```diff
@@ -27,11 +27,11 @@ import (
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
 
-	"github.com/mysteriumnetwork/go-openvpn/openvpn3"
 	"github.com/mysteriumnetwork/node/core/connection"
 	"github.com/mysteriumnetwork/node/core/ip"
 	"github.com/mysteriumnetwork/node/core/port"
 	"github.com/mysteriumnetwork/node/identity"
+	openvpn3 "github.com/mysteriumnetwork/node/mobile/mysterium/openvpn3"
```

Comment:
https://github.com/mysteriumnetwork/node/pull/2084

Reply:
You can revert it now, and I will check why it's not reverting back.

Reply:
my ide seems to be flipping them, i'll try and force a proper one